de.wagnermartin.Plattenalbum.json

```diff
@@ -0,0 +1,44 @@
+{
+  "app-id": "de.wagnermartin.Plattenalbum",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "45",
+  "sdk": "org.gnome.Sdk",
+  "command": "plattenalbum",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--share=network",
+    "--filesystem=xdg-music:ro",
+    "--filesystem=xdg-run/mpd",
+    "--talk-name=org.freedesktop.Notifications",
```

Comment:
IS this required? It should work with the portal with GTK4

Reply:
Apparently not anymore. Thanks for the tip!